src/courseware/course/CourseBreadcrumbs.jsx

```diff
@@ -14,6 +14,15 @@ const CourseBreadcrumb = ({
   content, withSeparator, courseId, sequenceId, unitId, isStaff,
 }) => {
   const defaultContent = content.filter(destination => destination.default)[0] || { id: courseId, label: '', sequences: [] };
+
+  let regularLinkRoute;
+  if (getConfig().ENABLE_LEGACY_NAV === 'true') {
```

Comment:
```suggestion
  if (getConfig().ENABLE_LEGACY_NAV) {
```

@ArturGaspar, let's change this to verify that the value is truthy. Currently, we need to explicitly set `ENABLE_LEGACY_NAV` to `'true'` (str) to enable it. Using `true` (bool) will not work. This seems to be quite inconsistent in the MFEs, but IMHO setting the truthy value as `'true'` is confusing.

I know we have the `getConfig().ENABLE_JUMPNAV !== 'true'` check in the same file, but that value is `'true'` by default, so it doesn't make such a big difference - if we want to disable it, we can set it to `false`, `'false'`, `''`, or `null` (or `0`, `1`, `'anything'`, though these are not typical use cases).

Reply:
@Agrendalath Done. Also on the upstream PR.